dream_layer_backend/extras.py

```
@@ -311,6 +312,101 @@ def upscale_image():
+        filename = data.get('filename', f'labeled_grid_{int(time.time())}.png')
+        show_labels = data.get('show_labels', True)
+        show_filenames = data.get('show_filenames', False)
+
+        # Parse grid size
+        if grid_size_str == 'auto':
+            grid_size = None
+        else:
+            try:
+                cols, rows = grid_size_str.split('x')
+                grid_size = (int(cols), int(rows))
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)